libs/core/graphql/src/apollo/get-registration/get-registration.ts

```diff
@@ -0,0 +1,12 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * A multi-provider injection token of the operation names of apollo queries that should 
+ * be GETs instead of POSTs. This is needed because apollo sends POST requests by default and daffodil
+ * defines all of the operation names. Each daffodil package will automatically register the requests
+ * that should be GETs when its driver package is loaded into the app. Simply filter on these provided
+ * request names for each outgoing request.
+ */
+export const DAFF_APOLLO_GET_REQUESTS = new InjectionToken<string>('DAFF_APOLLO_GET_REQUESTS', {
+	factory: () => ''
```

Comment:
Should this return an array?

Reply:
I'm not really sure what it should technically return. I thought that because it is a multiprovider, it will automatically be an array of whatever it is. It does work in the demo app as expected and comes through as an array of strings.

Reply:
upon further consideration, I decided it is better as an array. Changed

Reply:
IME, you should type multi tokens as strings